Xamarin.Forms.Xaml/ApplyPropertiesVisitor.cs

```diff
@@ -36,13 +36,15 @@ public ApplyPropertiesVisitor(HydrationContext context, bool stopOnResourceDicti
 		public bool StopOnResourceDictionary { get; }
 		public bool VisitNodeOnDataTemplate => true;
 		public bool SkipChildren(INode node, INode parentNode) => false;
-		public bool IsResourceDictionary(ElementNode node) => typeof(ResourceDictionary).IsAssignableFrom(Context.Types[node]);
+		public bool IsResourceDictionary(ElementNode node) => Context.Types.TryGetValue(node, out var type) && typeof(ResourceDictionary).IsAssignableFrom(type);
 
 		public void Visit(ValueNode node, INode parentNode)
 		{
 			var parentElement = parentNode as IElementNode;
 			var value = Values [node];
```

Comment:
Shouldn't this also be in a TryGetValue? We do that for the one in the next method: 
https://github.com/xamarin/Xamarin.Forms/pull/5369/files#diff-370e53f89e3c43f7073b0f2f85dcd6b2R100

Reply:
it's very unlikely for a ValueNode (usually created from a string literal) to fail construction, so this is probably fine